package_test.go

```diff
@@ -235,9 +234,55 @@ func TestDocumentsClientCreate(t *testing.T) {
 	}{}
 	require.NoError(t, json.Unmarshal([]byte(testResponse), expected))
 	assert.Equal(t, &expected.Data, got)
-}  
-  
-  
+}
+
+func TestAttachmentsClientCreate(t *testing.T) {
+	testResponse := exampleAttachmentsCreateResponse
+
+	// Prepare HTTP client with mocked transport.
```

Comment:
the `failed` test should also be there

Reply:
You mean I should add a tests were we check a failed scenario?

Reply:
Added the test here: https://github.com/ioki-mobility/go-outline/pull/41/commits/03c7e922d72e10d915766971f41024bbc316d0c4